ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Footer.kt

```diff
@@ -80,8 +91,7 @@ private fun Footer(
         modifier = Modifier
             .fillMaxWidth()
             .height(intrinsicSize = IntrinsicSize.Min)
-            .padding(horizontal = UIConstant.defaultHorizontalPadding)
-            .padding(bottom = UIConstant.defaultVerticalSpacing),
```

Comment:
Hmm so we are removing the bottom spacing even when not in full screen? Is the view affected?

Reply:
no, it looks better, there was too much space before

In `main` (with padding and no safe insets):

<img width="450" alt="Screenshot 2024-02-14 at 16 40 48" src="https://github.com/RevenueCat/purchases-android/assets/664544/09507703-475f-4e9d-a7ae-3ffd089fc547">


This branch without this change (with padding and safe insets):

| Full screen | No full screen |
|------------|---------------|
| <img width="450" alt="Screenshot 2024-02-14 at 15 53 00" src="https://github.com/RevenueCat/purchases-android/assets/664544/99f03068-d264-4b88-b4a2-6551090bda1a"> | <img width="448" alt="Screenshot 2024-02-14 at 15 53 59" src="https://github.com/RevenueCat/purchases-android/assets/664544/757c6cd9-d600-41f8-a63a-b23bb40b727d"> |

This branch with this change (no padding and safe insets):

| Full screen | No full screen |
|------------|---------------|
| <img width="450" alt="Screenshot 2024-02-14 at 15 50 54" src="https://github.com/RevenueCat/purchases-android/assets/664544/8fa28917-2fb1-4e42-b525-8bb9e0c3ce10"> | <img width="450" alt="Screenshot 2024-02-14 at 15 49 23" src="https://github.com/RevenueCat/purchases-android/assets/664544/d88b6630-d4d0-4e94-9f23-d56382e78689"> |

Reply:
Sounds good! we will need to update the screenshot tests then